### PR TITLE
[#9] Add skip clause

### PR DESCRIPTION
**Pull request by ejba**

### CODE REVIEW

File: src/main/java/com/dsl/clauses/links/AfterLimit.java [hunk 1]
```diff
@@ -2,5 +2,5 @@
 
 import com.dsl.AsString;
 
-public interface AfterLimit extends AsString, Returns, With {
+public interface AfterLimit extends AsString, Returns, With, Skip {
```
Comment (mcserra) [inline src/main/java/com/dsl/clauses/links/AfterLimit.java]: Skip after limit is invalid

Comment (mcserra) [reply src/main/java/com/dsl/clauses/links/AfterLimit.java]: skip after with is valid

File: src/main/java/com/dsl/clauses/links/AfterSkip.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.dsl.clauses.links;
+
+import com.dsl.AsString;
+
+public interface AfterSkip extends Returns, Limit, Skip, AsString {
```
Comment (mcserra) [inline src/main/java/com/dsl/clauses/links/AfterSkip.java]: missing with and match

**Review by mcserra — CHANGES_REQUESTED** [review]

File: src/main/java/com/dsl/clauses/links/AfterLimit.java (hunk 1 above)
Comment (ejba) [reply src/main/java/com/dsl/clauses/links/AfterLimit.java]: Solved, check again please!

File: src/main/java/com/dsl/clauses/links/AfterSkip.java (hunk 2 above)
Comment (ejba) [reply src/main/java/com/dsl/clauses/links/AfterSkip.java]: Solved, check again please!

**Review by mcserra — APPROVED** [review]

**Pull request merged**